packages/terra-consumer-nav/src/components/nav-help/NavHelpContent.jsx

```diff
@@ -51,7 +50,7 @@ class NavHelpContent extends React.Component {
             className={cx('help-item-wrapper')}
             key={content.text}
           >
-            <Button
+            <button
```

Comment:
terra-button v2 no longer accepts/renders `props.children`, switched to regular html `button` instead

Reply:
does it affect the style?

Reply:
Style looks the same between the two.

Side by side comparison:
<img width="962" alt="screen shot 2018-02-22 at 5 08 12 pm" src="https://user-images.githubusercontent.com/34115417/36569333-07f93548-17f3-11e8-9740-7263351b84e8.png">

Local changes:
![screen shot 2018-02-22 at 5 09 11 pm](https://user-images.githubusercontent.com/34115417/36569357-215114a2-17f3-11e8-903e-768fc291c71c.png)

Master:
![screen shot 2018-02-22 at 5 09 39 pm](https://user-images.githubusercontent.com/34115417/36569368-2e5431de-17f3-11e8-9693-d744c9fab3bb.png)